opensearch-development.yml

```diff
@@ -10,7 +10,7 @@ instance_groups:
   vm_type: t3.large
 
 - name: opensearch_dashboards
-  vm_type: t3.large
+  vm_type: t3.xlarge
```

Comment:
How do we know for sure that this is the problem with the Dashboards VM? I do see that the VM keeps dying, but I couldn't identify why

Reply:
its a maybe for now, i looked at the aws console and what we do for logsearch. in aws it shows less spikeage but logsearch only uses large. I ran it for 2 days or so with xlarge with no issue so it seems like a fix for now but if we see it not working we can roll it back.